source/manual/deploy-puppet.html.md

```diff
@@ -20,13 +20,19 @@ deploy.
 
 You can deploy Puppet to Staging and Production using the following steps:
 
-1. Deploy a newer version of govuk-puppet to staging by using the 'Deploy to Staging (Carrenza)'
-   and 'Deploy to Staging (AWS)' buttons in the Release app after clicking on the release
-   tag and reviewing the changes listed.
-1. You will either need to wait 30 minutes or read about [convergence](#convergence).
-   You should monitor Icinga and Smokey, and test anything you're concerned about.
+1. In the [Release app][release-app-puppet], click on the release tag and review
```

Comment:
To keep in line with GOV.UK style, replace "click on" with "select". This is because not all users "click", especially if they are using assistive tech.